spec/unit/network/http/handler_spec.rb

```diff
@@ -89,7 +95,7 @@ def respond(text)
       expect(res_body["issue_kind"]).to eq(Puppet::Network::HTTP::Issues::RUNTIME_ERROR.to_s)
       expect(res_body["message"]).to eq("Server Error: the sky is falling!")
       expect(res_body["stacktrace"].is_a?(Array) && !res_body["stacktrace"].empty?).to be_truthy
-      expect(res_body["stacktrace"][0]).to match("spec/unit/network/http/handler_spec.rb")
+      expect(res_body["stacktrace"][0]).to match(/The 'stacktrace' property is deprecated/)
```

Comment:
Should we have positive confirmation somewhere that the stack trace isn't included?